docs/deployment/deploy_with_existing_vineyard_cluster.md

```diff
@@ -36,12 +50,17 @@ python3 -m pip install vineyard
 ```
 
 By default, the Vineyard cluster consists of three Vineyard instances and three etcd instances. 
-However, since we only have one node in the Kubernetes cluster, we need to specify the number of Vineyard instances and etcd instances using the `vineyard_replicas` and `vineyard_etcd_replicas` parameters. 
+However, since we only have one node in the Kubernetes cluster, we need to specify the number of Vineyard instances and etcd instances using the `vineyard_replicas` and `vineyard_etcd_replicas` parameters. DON'T set the number of Vineyard instances and etcd instances to be greater than the number of nodes in the Kubernetes cluster.
+
 To deploy a simple Vineyard cluster with one Vineyard instance and one etcd instance, follow the next step:
 
 ```python
 import vineyard
 
+# The default namespace is `vineyard-system` and the default deployment name is `vineyardd-sample`. Also, the command will create the namespace if it doesn't exist.
+# For more details about the parameters, please refer to the doc of vineyardctl
+# https://github.com/v6d-io/v6d/blob/main/k8s/cmd/README.md
+# Notice, all character `-` in the parameter of vineyardctl should be replaced with `_` in the python API
 vineyard.deploy.vineyardctl.deploy.vineyard_deployment(
     vineyard_replicas=1, 
```

Comment:
> Only leave the `vineyard_replicas` is okay, the others could use the default. Also since you already use the name `vineyard_deployment`, why you prefix every parameters with `vineyard_`? for example, just `vineyard_deployment(replicas=1, etcd_replicas=2)` is meaningful enough for me.

Thanks for the feedback, I will change the parameters in the next version.